src/kbmod/run_search.py

```diff
@@ -41,7 +42,7 @@ def get_angle_limits(self, config):
         ang_max = config["average_angle"] + config["ang_arr"][1]
         return [ang_min, ang_max]
 
-    def do_gpu_search(self, config, search):
+    def do_gpu_search(self, config, search, strategy):
```

Comment:
I'd also just name this `trajectory_generator`, since a user could make a run method (eventually) be a logical loop like:

```
- sample a grid based generator
- run search, get results
- find most likely velocities/pixels/angles 
- resample with a finer grid generator
```
and now "strategy" becomes an overloaded term.